www/js/controller/home.controller.js

```diff
@@ -1,5 +1,19 @@
 angular.module('vamosJuntas').controller('HomeController',
  ['$scope', 'placeFactory', function($scope, placeFactory) {
- 	placeFactory.fetchPlaces();
- 	$scope.places = placeFactory.places;
+ 	$scope.places =	placeFactory.fetchPlaces().then(function (response) {
+    $scope.places = response.data;
+  });
+
+  $scope.getTotalOfOccurrences = function (place) {
+    var numberOfOccurrences = 0;
+    for(var i = 0; i < place.occurrences.length; i++) {
+      numberOfOccurrences += place.occurrences[i].numberOfOccurrences;
+    }
+    return numberOfOccurrences;
```

Comment:
Maybe we could use array.prototype.reduce to get the total number of occurrences here, what do you think?